src/api/cli/DaphneUserConfig.h

```diff
@@ -42,13 +42,15 @@ struct DaphneUserConfig {
     bool use_obj_ref_mgnt = true;
     bool use_ipa_const_propa = true;
     bool use_phy_op_selection = true;
+    bool use_mlir_codegen = false;
     bool cuda_fuse_any = false;
     bool vectorized_single_queue = false;
     bool prePartitionRows = false;
     bool pinWorkers = false;
     bool hyperthreadingEnabled = false;
     bool debugMultiThreading = false;
     bool use_fpgaopencl = false;
+    bool hybrid = false;
```

Comment:
The name hybrid alone is not very describing. What about mlir_hybrid_codegen?

Reply:
You'r right, I changed it as suggested.